app/src/main/res/layout/fragment_new_feature_release.xml

```diff
@@ -35,9 +35,10 @@
         android:layout_marginTop="20dp"
         app:cardCornerRadius="14dp"
         android:clickable="true"
-        app:cardElevation="8dp"
+        app:cardElevation="0dp"
```

Comment:
was not required to make it to `0dp` , could have just removed this line of code.